archetype/src/main/resources/META-INF/archetype-post-generate.groovy

```diff
@@ -15,6 +15,7 @@ def outputDirectory = new File(request.getOutputDirectory(), request.getArtifact
 validateInput(jakartaVersion, profile, javaVersion, runtime, docker, outputDirectory)
 generateRuntime(runtime, jakartaVersion, docker, outputDirectory)
 bindEEPackage(jakartaVersion, outputDirectory)
+generateWebapp(profile, outputDirectory)
```

Comment:
Please update the generated README to account for this change.

Reply:
Switched test URLs to Rest endpoint in case of core profile

Reply:
Thanks, that works.